moped-editor/src/views/auth/LoginView.js

```diff
@@ -43,16 +60,19 @@ const LoginView = () => {
         <Container maxWidth="sm">
           <Formik
```

Comment:
Here's Formik! Maybe we could swap out these login and register forms to be powered by React Hook Form, but they would perfectly fine as is. 👍